test/unit/SandboxCoordinatorSpec.ts

```diff
@@ -79,10 +79,30 @@ describe('SandboxCoordinator', () => {
   });
 
   describe('Given that maxConcurrentTestRunners config has been set', () => {
-    it('Then runMutants should use that config rather than cpuCount', (done) => {
+    it('if config is less than cpuCount, then runMutants should use that config rather than cpuCount', (done) => {
       const mutants: any[] = [mockMutant(0, []), mockMutant(1, [])];
 
       sut = new SandboxCoordinator({
+        maxConcurrentTestRunners: 1
+      }, expectedInputFiles, expectedTestFramework, reporter);
+
+      firstSandbox.runMutant.returns(Promise.resolve({ status: RunStatus.Complete, tests: [{ name: 'test1', status: TestStatus.Success }] }));
+      secondSandbox.runMutant.returns(Promise.resolve({ status: RunStatus.Complete, tests: [{ name: 'test1', status: TestStatus.Success }] }));
+
+      sut.runMutants(mutants)
```

Comment:
I would rather see a return statement here to return the result of  `runMutants.then()` (also a promise) instead of using the `done` parameter in this test. Using the `done` parameter is more error prone. For example, you forgot to handle the reject scenario of this promise. Now a rejection would result in a timeout eventually, not a useful failure message